_FIPS201/requirements.md

```diff
@@ -141,7 +141,7 @@ other authentication processes from [Section 6.2](authentication.md#s-6-2).
 The electronic facial image is a secondary means of authentication during operator-attended PIV issuance and maintenance processes
 when fingerprint biometric data records are unavailable.
 
-PIV background investigation, identity proofing, registration, and issuance processes **MAY** be performed across multiple sessions at different facilities. If multiple sessions are needed, the applicant **SHALL** be linked through a positive biometric verification decision by comparing biometric characteristics captured at a previous session with biometric characteristics captured during the current session. Issuers **SHALL** follow applicable federal laws and regulations regarding the retention and destruction of biometric data.
+PIV background investigation, identity proofing, registration, and issuance processes **MAY** be performed across multiple sessions at different facilities. If multiple sessions are needed, the applicant **SHALL** be linked through a positive biometric verification decision by comparing biometric characteristics captured at a previous session with biometric characteristics captured during the current session. Verifications **MAY** be performed against biometric characteristics stored on the PIV Card as described in this Section or against available biometric characteristics stored in the enrollment record. Fingerprints, facial image or iris (if implemented) **MAY** be used for verification. For linking background investiations, however, fingerprints only **MAY** be used, since fingerprints are the biometric characteristics used for background investigation. Issuers **SHALL** follow applicable federal laws and regulations regarding the retention and destruction of biometric data.
```

Comment:
I've reworked this a bit.

Part of this includes "electronically," with the goal being to avoid someone wanting to match against the face printed on the PIV card. Is this necessary?

```suggestion
PIV background investigation, identity proofing, registration, and issuance processes **MAY** be performed across multiple sessions at different facilities. If multiple sessions are needed, the applicant **SHALL** be linked through a positive biometric verification decision by comparing biometric representations captured at a previous session with biometric samples of fingerprints captured during the current session. Issuers **SHALL** follow applicable federal laws and regulations regarding the retention and destruction of biometric data.

For linking background investigations, only fingerprints **SHALL** be used, since fingerprints are the only biometric characteristic used for background investigations.  For all other purposes, verification attempts **MAY** be performed against any available biometric characteristic stored electronically on the PIV Card or in the enrollment record.
```

Reply:
@gfiumara:  I feel we are not responding to the comment anymore.  #211 states that we often seem to talk about fingerprints only and if there are other modality allowed.   I don't necessarily agree with that comment (see my initial position  to reject).  The group resolution was to mention the use of PIV card bio or enrollment record  bio,  where applicable, I believe... but that is lost now.   Let's talk tomorrow.  Granted, PIV card bio or enrollment record bio does not make sense when linking id proofing and enrollment (PIV card and enrollment record do not exist yet). It does make sense when linking to issuance. Maybe we should mention 'partial enrollement record', but that gets into too much detail. The more I think about it, the more I feel we should reject comment.  Section 2.5 has a good detailed list of uses of bio.  That should be enough for a high level document such as a FIPS.